README.md

```diff
@@ -1,63 +1,40 @@
-BinSkim Binary Analyzer
-=======================
+# BinSkim Binary Analyzer
```

Comment:
this file was there from the beginning. I just touched it up a bit. The User guide was the one I worked on a lot